templates/zerver/help/configure-mobile-notifications.md

```diff
@@ -0,0 +1,22 @@
+# Configure Mobile Push Notifications
+
+Follow these steps to change your mobile notification settings.
+
+{!go-to-the.md!} [Notifications](/#settings/notifications)
+{!settings.md!}
+
+    Under the **Private messages and @-mentions** section check the
```

Comment:
I don't know if you noticed, but these aren't step-numbered (1. 2. 3.) Did you miss these on accident?